backend/FwLite/MiniLcm/SyncHelpers/EntrySync.cs

```diff
@@ -130,6 +141,26 @@ public override Task<int> Replace(ComplexFormType before, ComplexFormType after)
         }
     }
 
+    private class PublicationsDiffApi(IMiniLcmApi api, Guid entryId) : ObjectWithIdCollectionDiffApi<Publication>
+    {
+        public override async Task<int> Add(Publication afterPub)
+        {
+            await api.AddPublication(entryId, afterPub.Id);
+            return 1;
+        }
+
+        public override async Task<int> Remove(Publication beforePub)
+        {
+            await api.RemovePublication(entryId, beforePub.Id);
+            return 1;
+        }
+
+        public override Task<int> Replace(Publication beforePub, Publication afterPub)
+        {
+            return Task.FromResult(0);
```

Comment:
I see you got this from copy-pasting.
I wonder if this is correct 🤔.
This is probably where FW-Lite would get changes made to publications in FW.

Reply:
This Replace method gets called when the `PublishIn` list in the `afterEntry` has a publication with the same ID as the list from the `beforeEntry`. I.e., all the time. If the publication *itself* has changed (i.e. different Name with same ID), then that's handled elsewhere (in PublicationSync). Here in EntrySync, we are only counting changes when an entry has a different set of publications than before.